### ISSUE TITLE
Modify state while device is off

### ISSUE THREAD

**Issue by radi-ka** [dresden-elektronik/deconz-rest-plugin#3603]
Hello there,

tldr: i want to be able to modify my lights state, while they are turned off. 

I'd like to build an external service that updates the brightness & color temp throughout the day, you know, warm and dark at night cold and bright in the morning, but with a smooth daily transition.

But to avoid "flicker" when my motion sensors trigger a light group they need to be already aware of their new state -> the state needs to be modifiable while the light is off.

The REST Api does not allow to set the brightness, while the light is off. ("... bri, is not modifiable. Device is set to off.")
So I tampered with the code a bit. I removed lines [973](https://github.com/dresden-elektronik/deconz-rest-plugin/blob/5d8aa096242dd69977ccf105cc22c3f816644a31/rest_lights.cpp#L973) through [980](https://github.com/dresden-elektronik/deconz-rest-plugin/blob/5d8aa096242dd69977ccf105cc22c3f816644a31/rest_lights.cpp#L980)
And... SUCCESS! the api responds with a success and the lights state is off but shows my updated value.
BUT: when i now send a simple "on" message, my state is ignored and the light turns on with bri=254

So i guess there is another state somewhere that is used here... :(

Any ideas how to solve my original problem in a different way or how to modify the state for good?
Or.. what am i missing?

thanks :)
